raiden/ui/cli.py

```diff
@@ -549,14 +549,14 @@ def app(
     blockchain_service = BlockChainService(
         privatekey_bin,
         rpc_client,
-        GAS_LIMIT,
         gas_price,
     )
 
     if sync_check:
         check_synced(blockchain_service)
 
-    discovery_tx_cost = gas_price * DISCOVERY_REGISTRATION_GAS
+    # 52100 gas is how much registerEndpoint() costs. Rounding to 60k for safety.
+    discovery_tx_cost = rpc_client.gasprice() * 60000
```

Comment:
In my opnion the named value is more understandable, why remove it?

Reply:
It was too high and removed it at first .. then at second pass I saw it's still needed for this error message and "hacked" it back in. I agree should be named and not just a constant.